charts/common/templates/lib/controller/_container.tpl

```diff
@@ -5,20 +5,27 @@ The main container included in the controller.
 - name: {{ include "common.names.fullname" . }}
   image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
   imagePullPolicy: {{ .Values.image.pullPolicy }}
+  {{- with .Values.command }}
+  command: {{ . }}
+  {{- end }}
   {{- with .Values.args }}
   args: {{ . }}
   {{- end }}
   {{- with .Values.securityContext }}
   securityContext:
     {{- toYaml . | nindent 4 }}
   {{- end }}
-  {{- if .Values.env }}
+  {{- if or .Values.env .Values.envTpl }}
```

Comment:
Not 100% sure about that, but this made me spot (I think) a small issue with the `{{- end }}` for this `{{ if }}` being in the wrong place

Reply:
Fixed the `{{ end }}` location.
Also just verified, since both `env` and `envTpl` are already set to an empty dict in the default values there won't be an issue with only one or the other being set.